src/templates/blogpost.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { graphql } from 'gatsby'
+import Layout from '../components/layout'
+import Block from '../components/block'
+import Break from '../components/break'
+import Prose from '../components/prose'
+
+
+export const BlogPostTemplate = ({
+  title,
+  body,
+  subtitle,
+  background,
+  audio,
+}) => (
+  <div>
+    <Block 
+      className={'bg-white'}
+    >
+      <Prose>
+        <div className={'pb-10'}>
+          <div className={'mb-5'}>
+            <a className={'text-blue-light text-md'} href={'/blog'}>View more articles</a>
+          </div>
+          <h1 className={'text-blue-dark text-4xl md:text-5xl leading-none font-bold mb-1'}>{title}</h1>
+          <div className={'text-xl md:text-2xl leading-tight mb-5'}>{subtitle}</div>
+          <Break />
+          <img src={background?.childImageSharp?.fluid?.src} className={'w-full mb-5'} />
+          {audio && (
+              <iframe height="52px" width="100%" title="Audio Player" frameborder="no" scrolling="no" seamless  src={audio}></iframe>
```

Comment:
we can probably remove the "scrolling" attribute here (or use `scrolling="auto"`) unless you've found this necessary?  I think this attribute is deprecated, and for users using magnification or zooming features in their browser this can sometimes make inner iframe content inaccessible